src/astt_gui/templates/index.html

```diff
@@ -561,7 +561,7 @@ <h4>Real Time Sensor Display</h4>
   <script src="static/js/echarts.min.js"></script>
   <script src="static/js/quill.min.js"></script>
```

Comment:
These lines and other similar lines makes reference to Javascript files.  If the  JS file is deleted, It's reference should also be deleted.